x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/transforms/DataFrameTransformTask.java

```diff
@@ -511,6 +511,16 @@ public synchronized boolean maybeTriggerAsyncJob(long now) {
                 return false;
             }
 
+            // ignore trigger if indexer is running, prevents log spam in A2P indexer
+            IndexerState indexerState = getState();
```

Comment:
#nit. 

This could all be in the same if statement
```
if (IndexerState.INDEXING.equals(indexerState) || IndexerState.STOPPING.equals(indexerState)) {
   logger.debug("Indexer for transform [{}] has state [{}], ignoring trigger", getJobId(), indexerState);
   return false;
}
```